[2023-07-01T00:00:23.304+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.inaccurate scheduled__2023-06-30T00:00:00+00:00 [queued]>
[2023-07-01T00:00:23.489+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.inaccurate scheduled__2023-06-30T00:00:00+00:00 [queued]>
[2023-07-01T00:00:23.498+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-01T00:00:23.749+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): inaccurate> on 2023-06-30 00:00:00+00:00
[2023-07-01T00:00:23.861+0000] {standard_task_runner.py:57} INFO - Started process 26406 to run task
[2023-07-01T00:00:23.896+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'scheduled__2023-06-30T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp4r3bl5mc']
[2023-07-01T00:00:23.923+0000] {standard_task_runner.py:85} INFO - Job 169: Subtask inaccurate
[2023-07-01T00:00:24.611+0000] {task_command.py:410} INFO - Running <TaskInstance: my_dag.inaccurate scheduled__2023-06-30T00:00:00+00:00 [running]> on host 414be138234a
[2023-07-01T00:00:26.276+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-30T00:00:00+00:00'
[2023-07-01T00:00:26.295+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-01T00:00:26.330+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo inaccurate']
[2023-07-01T00:00:26.399+0000] {subprocess.py:86} INFO - Output:
[2023-07-01T00:00:26.409+0000] {subprocess.py:93} INFO - inaccurate
[2023-07-01T00:00:26.414+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-01T00:00:26.593+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230630T000000, start_date=20230701T000023, end_date=20230701T000026
[2023-07-01T00:00:27.039+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-01T00:00:27.194+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
